### ISSUE TITLE
Loader doesn't resolve .ts files properly

### ISSUE THREAD

**Issue by ManuelSch** [xerebede/gridsome-plugin-typescript#12]
Hi,

whenever I try to import something from within a .vue file, `gridsome build` fails with the following error: 

```
yarn run v1.13.0
$ gridsome build
Gridsome v0.6.4

Initializing plugins...
Initialize - 0.96s
Load sources - 0s
Create GraphQL schema - 0.12s
Create pages and templates - 0.05s
Generate code - 0.03s
Bootstrap finish - 1.16s
Execute GraphQL (0 queries) - 0s
Write page data (0 files) - 0s
(undefined) ./src/pages/Index.vue?vue&type=script&lang=ts& (./node_modules/ts-loader??ref--15-0!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/pag
es/Index.vue?vue&type=script&lang=ts&)
Module not found: Error: Can't resolve '~/util/validate-email' in 'C:\Development\TestProjects\gridsome-ts\src\pages'
resolve '~/util/validate-email' in 'C:\Development\TestProjects\gridsome-ts\src\pages'
  Parsed request is a module
  using description file: C:\Development\TestProjects\gridsome-ts\package.json (relative path: ./src/pages)
    aliased with mapping '~': 'C:\Development\TestProjects\gridsome-ts\src' to 'C:\Development\TestProjects\gridsome-ts\src/util/validate-email'
      using description file: C:\Development\TestProjects\gridsome-ts\package.json (relative path: ./src/pages)
        Field 'browser' doesn't contain a valid alias configuration
        using description file: C:\Development\TestProjects\gridsome-ts\package.json (relative path: ./src/util/validate-email)
          no extension
            Field 'browser' doesn't contain a valid alias configuration
            C:\Development\TestProjects\gridsome-ts\src\util\validate-email doesn't exist
          .js
            Field 'browser' doesn't contain a valid alias configuration
            C:\Development\TestProjects\gridsome-ts\src\util\validate-email.js doesn't exist
          .vue
            Field 'browser' doesn't contain a valid alias configuration
            C:\Development\TestProjects\gridsome-ts\src\util\validate-email.vue doesn't exist
          ts
            Field 'browser' doesn't contain a valid alias configuration
            C:\Development\TestProjects\gridsome-ts\src\util\validate-emailts doesn't exist
          as directory
            C:\Development\TestProjects\gridsome-ts\src\util\validate-email doesn't exist
[C:\Development\TestProjects\gridsome-ts\src\util\validate-email]
[C:\Development\TestProjects\gridsome-ts\src\util\validate-email.js]
[C:\Development\TestProjects\gridsome-ts\src\util\validate-email.vue]
[C:\Development\TestProjects\gridsome-ts\src\util\validate-emailts]
 @ ./src/pages/Index.vue?vue&type=script&lang=ts& (./node_modules/ts-loader??ref--15-0!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/pages/Index.
vue?vue&type=script&lang=ts&) 21:0-50 30:15-28
 @ ./src/pages/Index.vue?vue&type=script&lang=ts&
 @ ./src/pages/Index.vue
 @ ./src/.temp/routes.js
 @ ./node_modules/gridsome/app/router.js
 @ ./node_modules/gridsome/app/app.js
 @ ./node_modules/gridsome/app/entry.client.js
 @ multi ./node_modules/gridsome/app/entry.client.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Process finished with exit code 1
```

This error is related to https://github.com/xerebede/gridsome-starter-typescript/issues/3

**Notice how it tries to resolve the .ts import:** `C:\Development\TestProjects\gridsome-ts\src\util\validate-emailts doesn't exist`. It seems the file extension for ts files is incorrect and should be ".ts" instead of "ts"

**Comment by ManuelSch** [comment]
I just noticed that there is a PR resolving this exact issue (https://github.com/xerebede/gridsome-plugin-typescript/pull/11). Would someone with write access please merge it and update the npm package?

**Comment by xerebede** [comment]
Fixed at PR #13.
